yapapi/strategy.py

```diff
@@ -38,7 +42,9 @@ class PropValueRange:
     def __contains__(self, item: float) -> bool:
         return (self.min is None or item >= self.min) and (self.max is None or item <= self.max)
 
-    def closest_acceptable(self, item: float):
+    def clamp(self, item: float):
+        if self.min is not None and self.max is not None and self.min > self.max:
+            raise ValueError(f"Cannot clamp to a range in which min={self.min} > max={self.max}.")
```

Comment:
I'd rather have this logic in `__post_init__`, because this class represents a "range" and if self.min > self.max then this is not a range.

Reply:
sigh... I went with your "empty range" idea... as I said, `__contains__` still makes sense...

Reply:
Imho this is a decision if we accept the range to be an empty range. Until now, we accepted. Your solution still accepts but refuses `clamp` on such range. I'd prefer a solution that disallows the empty range, but you current implementation is good enough.

Reply:
Maybe even your approach is better, let's leave it as it is now.

Reply:
honestly, I forgot it's a dataclass and thought that namedtuples don't have proper `__init__` afaik/r...